tests/Resources/ti.ui.view.test.js

```diff
@@ -1167,4 +1167,17 @@ describe('Titanium.UI.View', function () {
 		win.addEventListener('open', () => finish());
 		win.open();
 	});
+
+	it.android('filterTouchesWhenObscured', finish => {
+		const view1 = Ti.UI.createView();
+		should(view1.filterTouchesWhenObscured).be.false();
+		const view2 = Ti.UI.createView({ filterTouchesWhenObscured: true });
+		should(view2.filterTouchesWhenObscured).be.true();
+
+		win = Ti.UI.createWindow();
```

Comment:
Not sure if creating a window is necessary, since there's no additional validation?

Reply:
Yeah, I guess we don't need this. We always call the Java `setFilterTouchesWhenObscured()` method now when the view is created upon window open.